.github/workflows/full-tests-linux.yml

```diff
@@ -22,15 +22,22 @@ jobs:
         pip install pytest
         pip install pytest-cov
     - name: Get ephy_testing_data current head hash
-      id: vars
+      id: ephys
       run: echo "::set-output name=HASH_EPHY_DATASET::$(git ls-remote https://gin.g-node.org/NeuralEnsemble/ephy_testing_data.git HEAD | cut -f1)"
-    - uses: actions/cache@v2
+    - name: Cache ephys dataset - ${{ steps.ephys.outputs.HASH_EPHYS_DATASET }}
+      uses: actions/cache@v2
       id: cache-datasets
       with:
         path: /home/runner/work/nwb-conversion-tools/nwb-conversion-tools/ephy_testing_data
-        key: ecephys-dset-${{ steps.vars.outputs.HASH_EPHY_DATASET }}
-        restore-keys: ecephys-dset-${{ steps.vars.outputs.HASH_EPHY_DATASET }}
-    - name: Run pytest with coverage
+        key: ecephys-dset-${{ steps.ephys.outputs.HASH_EPHY_DATASET }}
+        restore-keys: ecephys-dset-${{ steps.ephys.outputs.HASH_EPHY_DATASET }}
+    - name: Set environment variables
+      run: |
+        echo "TEST_ENV=ci" >> $GITHUB_ENV
```

Comment:
So it looks from https://github.com/catalystneuro/nwb-conversion-tools/runs/4456719722?check_suite_focus=true that the "CI" key being used in `test_gin.py` is inherent to the workflow environment; thus the `TEST_ENV` variable here is going unused?

Do we need this line then?

Reply:
no we don't, we can actually clean everything I added in this file... I don't think i managed to add the ENV here, very frustrating, but at least it seems like the CI key should do it

Reply:
CI stuff is indeed quite frustrating sometimes. You did successfully add the environment variable, though.

![image](https://user-images.githubusercontent.com/51133164/145281732-5235d0f3-15c4-41e6-a4c4-e2a93f063fa7.png)

It's just that there's apparently a whole bunch of other latent environment variables I never knew about, and one of them is a boolean for if this is a CI or not, lol.